### ISSUE TITLE
Steam controller not detected in beta 1685572056

### ISSUE THREAD

**Issue by af7567** [ValveSoftware/steam-for-linux#9555]
#### Your system information

sysinfo: https://gist.github.com/af7567/116fce41c62c14d52ca4a72ece7cb9e0
* Steam client version (build number or date): 1685572056
* Distribution (e.g. Ubuntu): Slackware
* Opted into Steam client beta?: Yes

After today's update my steam controller is not recogised by Steam any more, it was working OK last night so seems to be something from this update. The controller connects via Bluetooth.

I tested with the stable steam client and it works OK still then, I see the following message in the terminal when turning the controller on:

```
Local Device Found
  type: 28de 1106
  path: /dev/hidraw8
  serial_number: c0:89:ac:fe:84:e7 - 0
  Manufacturer: 
  Product:      SteamController
  Release:      0
  Interface:    -1

!! Steam controller device opened for index 0.
```
Also get the steam popup saying that it is loading controller settings and the light on the controller dims since that is how I set it.

But in the beta client nothing happens at all and the controller just works as a mouse with the light staying on full brightness.

**Comment by af7567** [comment]
I just connected the controller by USB and it seems to work in wired mode OK with the beta client, so it's just Bluetooth mode that is broken since 1685572056.
I don't know where my original steam controller dongle is so I haven't been able to test with that.

**Comment by af7567** [comment]
Looks like this is fixed in 1686092969
